nservicebus/wcf/index.md

```diff
@@ -46,14 +67,21 @@ And finally expose the WCF service via the config file, for the example above th
 </system.serviceModel>
 ```
 
-The service name in `<service name="XXX"` needs to match the [`Type.FullName`](https://msdn.microsoft.com/en-us/library/system.type.fullname.aspx) that derives from `NServiceBus.WcfService<TCommand, TErrorCode>`
+The service name in `<service name="XXX"` needs to match the [`Type.FullName`](https://msdn.microsoft.com/en-us/library/system.type.fullname.aspx) that derives from `NServiceBus.WcfService<TRequest, TResponse>`
+
+partial:wcfbindingaddressincode
+
+partial:wcfrequestresponse
+
+partial:wcfcancellation
 
+partial:wcfrouting
 
 ## Queries and other return values
 
 To allow clients to perform queries, it is best not to use NServiceBus. Messaging is designed for non-blocking operations, and queries are (for the most part) operations for which the user wishes to wait.
 
-If there is some other operation that isn't strictly a query that returns a value for example some type of calculation consider invoking the operation locally where possible by referencing the DLL on the client.
+If there is some other operation that isn't strictly a query that returns a value, for example, some type of calculation consider invoking the operation locally where possible by referencing the DLL on the client.
```

Comment:
>If there is some other operation that isn't strictly a query that returns a value

How about:

When performing operations that aren't straightforward as a simple query to return a value, for example, a long calculation, consider invoking the operation locally where possible by referencing the DLL on the client.